package-lock.json

```diff
@@ -605,7 +605,7 @@
                 },
                 "readable-stream": {
                     "version": "2.0.6",
-                    "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-2.0.6.tgz",
+                    "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.0.6.tgz",
```

Comment:
These shouldn't change. See this comment on how to fix npm on your machine: https://github.com/npm/npm/issues/16938#issuecomment-339863980

Specifically the part about removing node modules and clearing the cache